src/Tizen.Peripheral/Tizen.Peripheral/Gpio.cs

```diff
@@ -0,0 +1,233 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using Tizen.Internals.Errors;
+using NativeGpio = Interop.Peripheral.Gpio;
+
+namespace Tizen.Peripheral
+{
+    /// <summary>
+    /// The class allows applications to use the platform Digital Pins as Input/Output.
+    /// </summary>
+    /// <privilege>http://tizen.org/privilege/peripheralio</privilege>
+    public class Gpio : IDisposable
+    {
+        /// <summary>
+        /// Enumeration of GPIO direction options.
+        /// </summary>
+        public enum Mode
+        {
+            /// <summary>
+            /// Input Mode.
+            /// </summary>
+            Input,
+
+            /// <summary>
+            /// Output mode with high value.
+            /// </summary>
+            OutputInitiallyLow,
+
+            /// <summary>
+            /// Output mode with low value.
+            /// </summary>
+            OutputInitiallyHigh,
+        }
+
+        /// <summary>
+        /// Enumeration of GPIO values.
+        /// </summary>
+        public enum Value
+        {
+            /// <summary>
+            /// Low value.
+            /// </summary>
+            Low = 0,
+
+            /// <summary>
+            /// High value.
+            /// </summary>
+            High
+        }
+
+        /// <summary>
+        /// Enumeration of edge types for the GPIO interrupt.
+        /// </summary>
+        public enum Edge
```

Comment:
How abut use [Flags]
https://docs.microsoft.com/ko-kr/dotnet/api/system.flagsattribute?view=netcore-3.1

``` c#
[Flags]
public enum Edge
{
      None = 0,
      Rising = 1,
      Falling = 2;
}
```

Reply:
I don't think it's needed.

Reply:
I followed Microsoft API and changed this enum to:
[GpioChangePolarity](https://docs.microsoft.com/en-us/uwp/api/windows.devices.gpio.gpiochangepolarity?view=winrt-19041)